test/lib/multiple.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const assert = require('chai').assert;
+const Fixture = require('../fixture');
+const queryAll = require('../../lib/utils/select').all;
+const simpleSnippet = require('../snippets/multiple.html');
+const Combobo = require('../../index');
+
+describe('multiselect config', () => {
```

Comment:
@vicaraya Thanks for the update. The only suggestion I have is to rename the file and the description to show that it is a test for initialization with `multiple` attribute and not about `multiselect` config. It might be a bit confusing for people I guess. My suggestion is:
1- Rename the files to "multiple-attribute"
2- Change the description here to "Initializing from select with `multiple` attribute".
3- Add a comment for line 15 (`simpleBox = new Combobo({ multiselect: false });`) and say. The select element has a `multiple` attribute that overrides the `multiselect` config. So it should be initialized as multi-select even if `multiselect` is set to false.

Reply:
All set, ty